### PR TITLE
Added manually merged "Vector Graphics Alpha" patch

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

File: loom/graphics/gfxVectorGraphics.h [hunk 1]
```diff
@@ -190,6 +190,7 @@ class VectorGraphics {
 	lmscalar boundT;
 	lmscalar boundR;
 	lmscalar boundB;
+    lmscalar alpha;
```
Comment (bengarney) [inline loom/graphics/gfxVectorGraphics.h]: indent?

**Comment by bengarney** [comment]
One nit, otherwise looks good.

**Comment by SmilyOrg** [comment]
Nitted?

**Comment by bengarney** [comment]
Looks good to me!

**Comment by bengarney** [comment]
Waiting on build.

**Pull request merged**